lib/services/ngsi/subscriptionService.js

```diff
@@ -245,6 +245,58 @@ function subscribeNgsi2(device, triggers, content, callback) {
         device, createSubscriptionHandlerNgsi2(device, triggers, store, callback));
 }
 
+
+/**
+ * Makes a subscription for the given device's entity using NGSIv2, triggered by the given attributes.
```

Comment:
```suggestion
 * Makes a subscription for the given device's entity using NGSIv2, triggered by the given attributes.
```

Still too many of them... an overall review should be done.

Reply:
Fixed a027234

Reply:
This one was not fixed at the end (my fault: my suggestion was not correct in my first comment). Now it should be ok:

```suggestion
 * Makes a subscription for the given device's entity using NGSI-LD, triggered by the given attributes.
```

Reply:
Fixed c70d9ed